presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2470,6 +2473,106 @@ public void testMismatchedBucketing()
         }
     }
 
+    @Test
+    public void testPartialMergePushdown()
```

Comment:
Could you please run it with and without materialization enabled? (see buckete execution tests)